test/integration/api/delete_account_secrets.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Load Environment vars
+source hack/scripts/test_envs 
+
+if [ -z "$OSD_STAGING_1_ACCOUNT_CR_NAME_OSD" ]; then
+    "OSD_STAGING_1_ACCOUNT_CR_NAME_OSD not set"
+    exit 1
+fi
+
+for secret in $(oc get secrets -n "${NAMESPACE}" | grep ${OSD_STAGING_1_ACCOUNT_CR_NAME_OSD} | awk '{ print $1 }')
```

Comment:
```suggestion
for secret in $(oc get secrets -n "${NAMESPACE}" | awk '/${OSD_STAGING_1_ACCOUNT_CR_NAME_OSD}/ { print $1 }')
```
Removes unnecessary `grep`.  Just a suggestion - not a big deal.  I think just a second ago, you were not using the `grep`, but had a duplicated $ in the `{print $$1}` command.  I was going to suggest removing that but Github said the diff was out of date.

Reply:
@clcollins Ha so I removed the second `$` and it wasn't working, I converted the quotes to `"` but still no love. Is your suggestion tested and working? Happy to do it if so! I was trying to do the right thing.

Reply:
Ah, you're right, I made a mistake in the suggestion.  This works:

`for secret in $(oc get secrets -n "${NAMESPACE}" | awk "/${OSD_STAGING_1_ACCOUNT_CR_NAME_OSD}/"'{ print $1 }')`

I think the second $ was actually breaking the awk printing, and printing the whole line, so it was deleting the secret by name, but also trying to delete secrets with names based on the other columns - working in spite of itself, heh.

Reply:
Thanks @clcollins updated the PR